src/OverviewPage/OverviewPage.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import React from "react";
+import { OverviewPage } from './OverviewPage'
+
+export default {
+    title: "Components/OverviewPage",
```

Comment:
```suggestion
    title: "Features/OverviewPage",
```
 Let's make this more prominent, it's not really a reusable component.